### PR TITLE
chore: YoStarKR ocr text update for SO LONG ADELE

### PR DESCRIPTION
**Pull request by HX3N**
add SL stage navigation and new character
include (#8018)

### CODE REVIEW

**Comment by HX3N** [comment]
@Constrat

File: resource/global/YoStarKR/resource/tasks.json [hunk 1]
```diff
@@ -51,24 +70,6 @@
...
-        "ocrReplace": [
-            [
-                ".*?전당",
-                "붉은노을전당"
-            ]
-        ],
-        "text": [
-            "붉은노을전당"
-        ]
-    },
```
Comment (Constrat) [inline resource/global/YoStarKR/resource/tasks.json:71]: This should not be removed (we are not in MaaRelease)

Comment (HX3N) [reply resource/global/YoStarKR/resource/tasks.json]: Oh, I see. I've restored it again.
I'm not sure, so I'm asking a question.
Do I need to update both MaaRelease and MaaAssistantArknights for Stage Navigation?

**Comment by HX3N** [comment]
Oh, I see. Thank you for letting me know.

**Pull request merged**